File: daySix/Javascript/QuestionOne.js
let checkNumberOfPrimeIn=(arrayOfNumbers)=>{
    let counter = 0;
    
    for(let index = 0; index < arrayOfNumbers.length; index++){
        let number = arrayOfNumbers[index];
        if(isPrime(number)){
            counter++;
        }
    }
    
    return counter;
}

let isPrime=(number)=>{
    let isPrime = false;
    let primeCounter = 0;
    
    for(let count = 1; count <= number; count++){
        if(number % count == 0){
            primeCounter++;
        }
    }
    
    if(primeCounter == 2){
        isPrime = true;
    }
    
    return isPrime;
}

let sortElementsIn=(arrayOfNumbers)=>{
    let store = 0;
    
    for(let index = 0; index < arrayOfNumbers.length; index++){
        for(let count = 0; count < arrayOfNumbers.length; count++){
            if(arrayOfNumbers[index] < arrayOfNumbers[count]){
                store = arrayOfNumbers[index];
                arrayOfNumbers[index] = arrayOfNumbers[count];
                arrayOfNumbers[count] = store;
            }
        }
    }
    
    return arrayOfNumbers;
}

let getPrimesIn=(arrayOfNumbers)=>{
    let arrayOfPrimes = [];
    
    for(let index = 0; index < arrayOfNumbers.length; index++){
        let number = arrayOfNumbers[index];
        
        if(isPrime(number)){
            arrayOfPrimes.push(arrayOfNumbers[index]);
        }
    }
    
    return arrayOfPrimes;
}


arrayOfNumbers = [5,4,3,9,2];

console.log(getPrimesIn(arrayOfNumbers));

console.log(sortElementsIn(getPrimesIn(arrayOfNumbers)));

File: daySix/python/question_one.py
def check_number_of_prime_in(array_of_numbers):
    counter = 0
    
    for index in range(len(array_of_numbers)):
        number = array_of_numbers[index];
        if(is_prime(number)):
            counter += 1
    
    return counter


def is_prime(number):
    is_prime = False
    prime_counter = 0
    
    for count in range(1, number + 1):
        if(number % count == 0):
            prime_counter += 1
    
    if(prime_counter == 2):
        is_prime = True
   
    return is_prime


def sort_elements_in(array_of_numbers):
    store = 0
    
    for index in range (len(array_of_numbers)):
        for count in range (len(array_of_numbers)):
            if(array_of_numbers[index] < array_of_numbers[count]):
                store = array_of_numbers[index]
                array_of_numbers[index] = array_of_numbers[count]
                array_of_numbers[count] = store
    
    return array_of_numbers


def get_primes_in(array_of_numbers):
    array_of_primes = []
    counter = 0
    
    
    for index in range (len(array_of_numbers)):
        number = array_of_numbers[index]
        
        if(is_prime(number)):
            array_of_primes.append(array_of_numbers[index])
    
    return array_of_primes
    
    
    
    

